exercises/luhn/src/example.erl

```diff
@@ -34,7 +47,7 @@ checksum([H | ReversedNumber], even, Total) when H >= $5 ->
 
 
 valid(Number) ->
-  checksum(Number) rem 10 == 0.
+  isNumeric(Number) andalso checksum(Number) rem 10 == 0 andalso length(isMinLength(Number)) > 1.
 
```

Comment:
This iterates multiple times over the input.

Validation of the input should be doable while calculating the checksum.